strategies/cctp_lend/deploy/src/artifacts/neutron_strategy_config.toml

```diff
@@ -0,0 +1,22 @@
+grpc_url = "http://rpc.neutron.quokkastake.io"
+grpc_port = "9090"
+chain_id = "neutron-1"
+min_ibc_fee = "1"
```

Comment:
hmm, just realized this is hardcoded to `Uint128::one()` in some places, will address in a follow up pr